x-pack/docs/en/rest-api/ml/resultsresource.asciidoc

```diff
@@ -405,6 +405,13 @@ A category resource has the following properties:
 `examples`::
   (array) A list of examples of actual values that matched the category.
 
+`grok_pattern`::
+  (string) A Grok pattern that could be used in Logstash or an Ingest Pipeline
+  to extract fields from messages that match the category. This field is
+  experimental and may be changed or removed in a future version. The Grok
```

Comment:
@lcawl do you know if there are any precedents for a single field that is experimental within a feature that is generally fully supported?  Is the way I've documented it here right?

Reply:
@droberts195 I'd suggest putting "experimental[]" after the data type, per https://github.com/elastic/docs#experimental-and-beta